### DISCUSSION TITLE
Use nested menus for menu entry swapper in the bank

### DISCUSSION THREAD

**Discussion by benjl** [runelite/runelite#15901] [Ideas]
The shift right click menu in the bank has a ton of entries (Deposit 1, 5, 10, x, all, reset, Withdraw 1, 5, 10, x, all, eat/wield) and would benefit greatly from the new nested menus.

**Comment by Nightfirecat** [comment]
I've created #15917 which, if merged, will add submenu functionality for bank and UI swaps.
